Lending API improvements (#791)
codatio/codat-docs: docs/lending/get-started.md

import { IntegrationsList } from "@components/global/Integrations";
import { accountingIntegrations, bankingIntegrations, commerceIntegrations } from "@components/global/Integrations/integrations";

:::tip Your lending journey

> Our Lending API supports the data collection step of your lending journey, which starts in your own web application. Enable Lending API and configure it, then utilize our authorization component in your app. Determine where the collected data will be stored and handle the subsequent steps of the lending process in your app. 

Reviewer: I don't think we've ever said 'authorization component' before? Can we say embedded link SDK or something, and add a link to the relevant page?

### Authorization flow

As part of using the Lending API, you will need your customers to authorize your access to their data. To do so, use Link - our pre-built, conversion-optimized, and white-labelled authorization flow. 

> We recommend you fully embed the Link auth flow in your experience by using our [Embedded Link](/auth-flow/authorize-embedded-link) SDK in your front-end code. You can also choose our out-of-the-box [Hosted Link](/auth-flow/authorize-hosted-link) auth flow option to get up and running as quick as possible. 

Reviewer: Front-end or frontend

## Use Lending API

> As a prerequisite for data collection, you need to create a Codat company and connect it to a data source (for example, an accounting platform). The company represents your SMB customer's business that is sharing access to their data. 

Reviewer: ```suggestion
Before you can collect your SMB customer's data, you need to create a Codat company and connect it to a data source (for example, an accounting platform). The company represents your customer's business that is sharing access to their data. 
```
Style-wise, I would prefer less formal language here

Also if we merge in the glossary PR then that might help here?

You can create a company:

* In the [Codat Portal](https://app.codat.io) by navigating to **Companies > Create company**
* By calling the [Create company](/lending-api#/operations/create-company) endpoint of our API

Remember to [authenticate](/using-the-api/authentication) if you are making calls to our API. Navigate to **Developers > API keys** in the Portal to pick up your authorization header.

> To establish a connection to a data source and sync business data, your customer must grant you access. They can do so using our [Link auth flow](/auth-flow/overview) solution, which we recommend you [use in your app](/lending/get-started#authorization-flow).

Reviewer: Link auth flow - make consistent with the language early

> Once the connection is established, Codat will perform the initial syncs for the data types you have previously enabled. You can use the `New company synchronised` [webhook](/using-the-api/webhooks/core-rules-types#new-company-synchronized) to get notified once these initial syncs are complete, and at least one of them is successful.

Reviewer: Enabled isn't a thing

```suggestion
Once the connection is established, Codat will retrieve data based of your data type settings (where you have previously set 'fetch on first link'). You can use the `New company synchronised` [webhook](/using-the-api/webhooks/core-rules-types#new-company-synchronized) to get notified once these initial syncs are complete, and at least one of them is successful.
```
